lib/awful/titlebar.lua

```diff
@@ -754,14 +766,20 @@ end
 -- @constructorfct awful.titlebar.widget.button
 function titlebar.widget.button(c, name, selector, action)
     local ret = imagebox()
-
     if titlebar.enable_tooltip then
-        ret._private.tooltip = atooltip({ objects = {ret}, delay_show = 1 })
-        ret._private.tooltip:set_text(name)
+        ret._private.tooltip = atooltip({
+            objects = {ret},
+            delay_show = 1.0,
+            margins_leftright = 12,
```

Comment:
Can you add 4 `beautiful` variables for these and leave the default as they were before (`nil`). This way it doesn't cause a change in behavior, but can still be defined.

I know there is a lot of precedents to do what you did, but we try not to add more.

Reply:
Done. Add theme variables and documentation.